nms/app/packages/magmalte/app/components/lte/LteSections.js

```diff
@@ -39,7 +39,9 @@ import SettingsInputAntennaIcon from '@material-ui/icons/SettingsInputAntenna';
 import ShowChartIcon from '@material-ui/icons/ShowChart';
 import SubscriberDashboard from '../../views/subscriber/SubscriberOverview';
 import Subscribers from '../Subscribers';
+import TracingDashboard from '../../views/tracing/TracingDashboard';
 import TrafficDashboard from '../../views/traffic/TrafficOverview';
+import WatchIcon from '@material-ui/icons/Watch';
```

Comment:
Can you use the one specified in call tracing figma instead ?  I believe it is this one
import LineStyleIcon from '@material-ui/icons/LineStyle';

Reply:
Okay